src/server/systems/NetworkingServer.cpp

```diff
@@ -1,16 +1,4 @@
 #include "NetworkingServer.hpp"
-#include <atomic>
-#include <memory>
-#include <utility>
-#include <vector>
-
-#include "../../common/events/ClientConnectedEvent.hpp"
-#include "../../common/events/ClientDisconnectedEvent.hpp"
-#include "../../common/events/EventController.hpp"
-#include "../../common/events/PlayerCreatedEvent.hpp"
-#include "../../common/util/InputDef.hpp"
-#include "../../common/util/game_settings.hpp"
-#include "../../common/util/network_util.hpp"
```

Comment:
why :o

my intuition was to include these here in instead of the hpp because the hpp doesn't really use them

Maybe your change even makes this slower because the hpp file you moved these into is included somewhere else, it'll bring along all those dependencies (i dont know cpp well enough to be certain tho)
youknowhatimean?

Reply:
Maybe revert 038960702e19489d2e88a68f118649797b838377
doesn't seem like you used that change